service/service.go

```diff
@@ -228,10 +228,30 @@ func (svc *Service) Close(ctx context.Context) error {
 	return nil
 }
 
-func registerCheckers(ctx context.Context, hc HealthChecker, dpESClient dpEsClient.Client) (err error) {
-	if err = hc.AddCheck("Elasticsearch", dpESClient.Checker); err != nil {
+func registerCheckers(ctx context.Context, hc HealthChecker, clList *api.ClientList) (err error) {
+	if err = hc.AddCheck("Berlin-API", clList.BerlinClient.Checker); err != nil {
+		log.Error(ctx, "error creating elasticsearch health check", err)
+		err = errors.New("Error(s) registering checkers for health check")
+		return err
+	}
+
+	if err = hc.AddCheck("Category-API", clList.ScrubberClient.Checker); err != nil {
 		log.Error(ctx, "error creating elasticsearch health check", err)
 		err = errors.New("Error(s) registering checkers for health check")
+		return err
 	}
-	return err
+
+	if err = hc.AddCheck("Elasticsearch", clList.DpESClient.Checker); err != nil {
+		log.Error(ctx, "error creating elasticsearch health check", err)
+		err = errors.New("Error(s) registering checkers for health check")
+		return err
+	}
+
+	if err = hc.AddCheck("Scrubber-API", clList.ScrubberClient.Checker); err != nil {
+		log.Error(ctx, "error creating elasticsearch health check", err)
+		err = errors.New("Error(s) registering checkers for health check")
+		return err
+	}
+
```

Comment:
All these logs are the same - they also don't appear to be feature flagged. Additionally - it's going through the router at the moment so I don't think it'll actually be able to pick up the health checks through there. Probably best if this is left out for now.

Reply:
I'll remove them